core/admin.py

```diff
@@ -48,12 +50,122 @@ class MessageAdmin(admin.ModelAdmin):
         "subject",
         "account",
     )
+
+    """
```

Comment:
What is this docstring for? Should it be removed?

Reply:
Heh. I was using it to copy and paste. Removed.